### PR TITLE
Fix some typos

### PR DESCRIPTION
**Pull request by mhmdanas**
#### What is it?
- [ ] Bugfix (user facing)
- [ ] Feature (user facing)
- [ ] Codebase improvement (dev facing)
- [X] Meta improvement to the project (dev facing)

#### Description of the changes in your PR

Just some typo fixes.

#### APK testing 

[debug.zip](https://github.com/TeamNewPipe/NewPipe/files/5649032/debug.zip)

#### Due diligence
- [x] I read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md).

### CODE REVIEW

File: app/src/main/java/org/schabi/newpipe/util/KioskTranslator.java [hunk 1]
```diff
@@ -5,7 +5,7 @@
 import org.schabi.newpipe.R;
 
 /**
- * Created by Chrsitian Schabesberger on 28.09.17.
+ * Created by Christian Schabesberger on 28.09.17.
```
Comment (mhmdanas) [inline app/src/main/java/org/schabi/newpipe/util/KioskTranslator.java:8]: Is this allowed?

Comment (TobiGr) [reply app/src/main/java/org/schabi/newpipe/util/KioskTranslator.java:8]: yes, see also #4961

**Comment by TobiGr** [comment]
I merged #4961. Please rebase :)

**Comment by mhmdanas** [comment]
Done.

**Review by TobiGr — APPROVED** [review]

**Comment by TobiGr** [comment]
thanks

**Pull request merged**